reports/Ohio.md

```diff
@@ -1,3 +1,16 @@
+## Canton
+
+### Police spray mace, fire tear gas and flash bangers at protestors, even after they have dispersed | May 29th
```

Comment:
Huh hadn't heard flash bangers as a way to describe those...

Reply:
Yeah I think this should change to flashbang 😬